File: app/src/main/java/com/packg/easynotes/Elements/Event.java
package com.packg.easynotes.Elements;

import java.util.Date;

public class Event {
    private Date date;
    private String eventText;

    public Event(Date date, String eventText){
        this.date = date;
        this.eventText = eventText;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getEventText() {
        return eventText;
    }

    public void setEventText(String eventText) {
        this.eventText = eventText;
    }
}

File: app/src/main/java/com/packg/easynotes/Elements/Event.kt
package com.packg.easynotes.Elements

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.util.*

@Entity(tableName = "event_table")
class Event(@ColumnInfo(name = "date") val date : Date,
            @ColumnInfo(name = "text") val text : String) {

    @ColumnInfo(name = "id")
    @PrimaryKey(autoGenerate = true) var id: Long = 0
}